payjoin/src/core/receive/mod.rs

```diff
@@ -405,12 +454,13 @@ mod tests {
             outpoint,
             witness_script.clone(),
             Some(sequence),
-        )
-        .unwrap();
-        assert_eq!(p2wsh_pair.txin.previous_output, outpoint);
-        assert_eq!(p2wsh_pair.txin.sequence, sequence);
-        assert_eq!(p2wsh_pair.psbtin.witness_utxo.unwrap(), p2wsh_txout);
-        assert_eq!(p2wsh_pair.psbtin.witness_script.unwrap(), witness_script);
+        );
+
+        // We do not support P2wsh yet
```

Comment:
This comment seems a bit out of place. We do support p2wsh because of this PR. I believe if we resolve [this comment](https://github.com/payjoin/rust-payjoin/pull/772/#discussion_r2171968524) we should be able to revert this change . 
And have a seperate test condition for when we use the general input pair new constructor for a p2wsh input and no expect weight param